### PR TITLE
[OTA-2356] Use messages for assignments

### PR DESCRIPTION
**Pull request by nkly**
Depends on advancedtelematic/libats#95

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -15,7 +15,7 @@ libraryDependencies ++= {
   val scalaTestV = "3.0.5"
   val bouncyCastleV = "1.57"
   val tufV = "0.6.0-1-g2f23c85"
-  val libatsV = "0.2.1-13-g4a34692"
+  val libatsV = "0.2.1-4-gd9e0738"
```
Comment (houcros) [inline build.sbt]: Reverting to an older version of libats?

Comment (nkly) [reply build.sbt]: It's just a snapshot that I published locally. Will be updated when we publish a version of libats from advancedtelematic/libats#95

File: src/main/scala/com/advancedtelematic/director/http/AdminResource.scala [hunk 2]
```diff
@@ -254,12 +252,7 @@ class AdminResource(extractNamespace: Directive1[Namespace], val keyserverClient
           queueForDevice(ns, device)
         } ~
         (path("cancel") & put) {
-          val f = cancelUpdate.one(ns, device).flatMap{ res =>
-            messageBusPublisher
-              .publish(UpdateSpec(ns, device, UpdateStatus.Canceled))
-              .map(_ => res.device)
-          }
-          complete(f)
+          complete(cancelUpdate.one(ns, device).map(_.device))
```
Comment (houcros) [inline src/main/scala/com/advancedtelematic/director/http/AdminResource.scala]: Should we add another `Deprecated in favor of "assignments"` comment here?

Comment (nkly) [reply src/main/scala/com/advancedtelematic/director/http/AdminResource.scala]: Yeah, good point

**Comment by nkly** [comment]
Knock-knock @koshelev @jerrytrieu. It would be nice if you could check it out.

File: src/main/scala/com/advancedtelematic/director/db/SetMultiTargets.scala [hunk 3]
```diff
@@ -94,18 +94,31 @@ class SetMultiTargets()(implicit messageBusPublisher: MessageBusPublisher) exten
...
-          _ <- messageBusPublisher.publish(UpdateSpec(namespace, device, UpdateStatus.Pending))
-          deviceUpdateEvent: DeviceUpdateEvent = DeviceUpdateAssigned(
-            namespace,
-            Instant.now(),
-            correlationId,
-            device)
-          _ <- messageBusPublisher.publish(deviceUpdateEvent)
-        } yield ()
-      }.map(_ => scheduled)
+    def publishDeviceScheduledMsg(device: DeviceId): Future[Unit] = for {
```
Comment (jerrytrieu) [inline src/main/scala/com/advancedtelematic/director/db/SetMultiTargets.scala]: Let's say `Assigned` instead of `Scheduled`

**Comment by jerrytrieu** [comment]
Let's put these changes on berlin hat and run e2e.

**Pull request closed without merging**